### PR TITLE
OSDOCS-10856: Add release note for bpfman Operator

### PR DESCRIPTION
**Pull request by jab-rh**
- https://issues.redhat.com/browse/OSDOCS-10856

Version(s): 4.17+

Issue:

Link to docs preview: https://79943--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-ebpf-program-support-ingress-node-firewall-operator_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1162,6 +1166,16 @@ In the following tables, features are marked with the following statuses:
 |====
 |Feature |4.15 |4.16 |4.17
 
+|Ingress Node Firewall Operator
+|General Availability
+|General Availability
+|General Availability
+
+|eBPF Operator
```
Comment (msherif1234) [inline release_notes/ocp-4-17-release-notes.adoc:1174]: what is eBPF operator ? you probably meant `bpfman-operator` ?

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -513,6 +513,10 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 ==== CoreDNS update to version 1.11.3
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
+[id="ocp-4-17-ebpf-program-support-ingress-node-firewall-operator_{context}"]
+==== eBPF program support for Ingress Node Firewall Operator (Tech Preview)
+
+The bpfman Operator, available as a Technology Preview, allows you to deploy and manage eBPF programs. It facilitates the loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information on deploying the bpfman Operator, see xref:../networking/network_security/bpfman-operator/bpfman-operator-about.adoc#bpfman-operator-about[About the bfpman Operator].
```
Comment (msherif1234) [inline release_notes/ocp-4-17-release-notes.adoc:519]: we should mention something about security or loading eBPF in secure way ?

Comment (Billy99) [reply release_notes/ocp-4-17-release-notes.adoc:519]: +1

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -561,6 +561,16 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-ebpf-manager-operator-technology-preview_{context}"]
+==== eBPF manager Operator (Tech Preview)
+
+The eBPF manager Operator, available as a Technology Preview, allows you to securely deploy and manage eBPF programs. It facilitates the secure loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information on deploying the bpfman Operator, see xref:../networking/network_security/bpfman-operator/bpfman-operator-about.adoc#bpfman-operator-about[About the eBPF manager Operator].
+
+[id="ocp-4-17-ebpf-program-support-ingress-node-firewall-operator_{context}"]
+==== eBPF program support for Ingress Node Firewall Operator (Tech Preview)
```
Comment (tmalove) [inline release_notes/ocp-4-17-release-notes.adoc:570]:
```suggestion
==== eBPF program support for Ingress Node Firewall Operator (Technology Preview)
```
I can't find the reference doc for this, but I am pretty sure the name is spelled out.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 4]
```diff
@@ -1380,6 +1390,16 @@ In the following tables, features are marked with the following statuses:
...
+|General Availability
+|General Availability
+|General Availability
+
+|eBPF manager Operator
+|N/A
+|N/A
+|Technology Preview
+
 |Advertise using L2 mode the MetalLB service from a subset of nodes, using a specific pool of IP addresses
```
Comment (tmalove) [inline release_notes/ocp-4-17-release-notes.adoc:1403]:
```suggestion
|Advertise by using L2 mode the MetalLB service from a subset of nodes, using a specific pool of IP addresses
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

File: release_notes/ocp-4-17-release-notes.adoc [hunk 5]
```diff
@@ -561,6 +561,16 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-ebpf-manager-operator-technology-preview_{context}"]
+==== eBPF manager Operator (Tech Preview)
+
+The eBPF manager Operator, available as a Technology Preview, allows you to securely deploy and manage eBPF programs. It facilitates the secure loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information on deploying the bpfman Operator, see xref:../networking/network_security/bpfman-operator/bpfman-operator-about.adoc#bpfman-operator-about[About the eBPF manager Operator].
```
Comment (tmalove) [inline release_notes/ocp-4-17-release-notes.adoc:567]: I get a 'page not found' error for the link for 'About the eBPF manager Operator.

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-17-release-notes.adoc (hunk 5 above)
Comment (jab-rh) [reply release_notes/ocp-4-17-release-notes.adoc:567]: @tmalove, yeah, I need to merge that feature PR, and then this should work.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 6]
```diff
@@ -561,6 +561,16 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-ebpf-manager-operator-technology-preview_{context}"]
+==== eBPF manager Operator (Tech Preview)
+
+The eBPF manager Operator, available as a Technology Preview, allows you to securely deploy and manage eBPF programs. It facilitates the secure loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information about deploying the bpfman Operator, see xref:../networking/network_security/ebpf_manager/ebpf-manager-operator-about.adoc#bpfman-operator-about[About the eBPF manager Operator].
```
Comment (Billy99) [inline release_notes/ocp-4-17-release-notes.adoc:567]:
```suggestion
The eBPF manager Operator, available as a Technology Preview, allows you to securely deploy and manage eBPF programs. It facilitates the secure loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information about deploying the bpfman Operator, see xref:../networking/network_security/ebpf_manager/ebpf-manager-operator-about.adoc#ebpf-manager-operator-about[About the eBPF manager Operator].
```
This is probably why CI failed.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 7]
```diff
@@ -561,6 +561,16 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-ebpf-manager-operator-technology-preview_{context}"]
+==== eBPF manager Operator (Tech Preview)
```
Comment (anfredette) [inline release_notes/ocp-4-17-release-notes.adoc:565]: Capitalize "M" "Manager" throughout.

**Review by anfredette — APPROVED** [review]
/lgtm

**Comment by kalexand-rh** [comment]
@anuragthehatter, will you please confirm this release note?

**Review by kalexand-rh — APPROVED** [review]

**Pull request merged**